### PR TITLE
feat: add `XChainBridge` info on door accounts' Account pages

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR adds a section to the Account Header of a page (next to the Signer List) with bridge details, if the account is a door account, and adds a little banner at the top. It also redoes the CSS a bit for the signer list.

### Context of Change

Better presentation of bridge info

### Type of Change

- [x] New feature (non-breaking change which adds functionality)

### TypeScript/Hooks Update

N/A

### Door Account
Before:
image

After:
Monosnap XRPL Explorer | rMAXACCrp3Y8  2023-06-01 12-14-13

### Non-door account
Before:
image

After:
Monosnap XRPL Explorer | r481fwr9G6Gq  2023-06-01 12-22-00

## Test Plan

CI passes. Works locally.

### CODE REVIEW

File: src/containers/Accounts/AccountHeader/styles.scss [hunk 1]
```diff
@@ -3,10 +3,44 @@
...
+      flex-direction: row;
+      align-items: center;
+    }
+
+    h1 {
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+
+    .door-account {
```
Comment (ckniffen) [inline src/containers/Accounts/AccountHeader/styles.scss:29]: This has a lot of shared styles from the purple "NFT" badge.  Really the only difference should be color.  Something like this is often called a `.badge` or `.pill`
https://livenet.xrpl.org/nft/00092710F86687692510076EFAD3F5C0416552A25801D8F410E83C9C00006201

File: src/containers/Accounts/AccountHeader/SubheaderLine.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { PropsWithChildren } from 'react'
+import clsx from 'clsx'
```
[the remark this answers is not in the document]
Comment (mvadari) [reply src/containers/Accounts/AccountHeader/SubheaderLine.tsx]: https://github.com/ripple/explorer/pull/724

**Review by ckniffen — APPROVED** [review]

File: src/rippled/lib/rippled.js [hunk 3]
```diff
@@ -215,6 +216,50 @@ const getAccountPaychannels = async (
...
+    ledger_index,
+    type: 'bridge',
+    limit: 400,
+  }).then((resp) => {
+    if (resp.error === 'actNotFound') {
+      throw new Error('account not found', 404)
+    }
+    if (resp.error === 'invalidParams') {
+      // thrown when XChainBridge amendment is not activated
+      // TODO: remove this when XLS-38d is live in mainnet
```
Comment (justinr1234) [inline src/rippled/lib/rippled.js:233]: Is there a ticket for this?

**Review by justinr1234 — APPROVED** [review]

File: src/rippled/lib/rippled.js (hunk 3 above)
Comment (mvadari) [reply src/rippled/lib/rippled.js:233]: Yes

**Pull request merged**